packages/trpc/server/routers/viewer/bookings/delete.handler.ts

```diff
@@ -0,0 +1,79 @@
+import { prisma } from "@calcom/prisma";
+import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
+
+import type { TDeleteInputSchema } from "./delete.schema";
+
+type DeleteOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TDeleteInputSchema;
+};
+
+export const deleteHandler = async ({ ctx, input }: DeleteOptions) => {
+  const { id } = input;
+  const { user } = ctx;
+
+  const booking = await prisma.booking.findFirst({
+    where: {
+      id: id,
+      OR: [
+        {
+          userId: user.id,
+        },
+        {
+          attendees: {
+            some: {
+              email: user.email,
+            },
+          },
+        },
+      ],
+    },
+    include: {
+      attendees: true,
+      references: true,
+      payment: true,
+      workflowReminders: true,
+      eventType: {
+        include: {
+          team: {
+            include: {
+              members: true,
+            },
+          },
+        },
+      },
+    },
+  });
+
+  if (!booking) {
+    throw new Error("Booking not found");
+  }
+
+  const isTeamEvent = booking.eventType?.team;
```

Comment:
This is not actually a boolean, why not use this variable to interact with the team?

Reply:
by interact with team, what do you mean?, I am sorry, I don't undertand that.